e2e/e2e-local.sh

```diff
@@ -2,16 +2,8 @@
 
 set -euxo pipefail
 
-: "${BUILD_ID:=local}"
-: "${SUBSCRIPTION_ID:=8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8}" #Azure Container Service - Test Subscription
-: "${LOCATION:=eastus}"
-: "${AZURE_TENANT_ID:=72f988bf-86f1-41af-91ab-2d7cd011db47}"
 : "${TIMEOUT:=90m}"
 : "${PARALLEL:=100}"
 
-export SUBSCRIPTION_ID
-export LOCATION
-export AZURE_TENANT_ID
```

Comment:
I like my debugger. I want to run tests without shell script.

So, I moved it as defaults in go config

Reply:
Could we create a task to ensure we create a Wiki page/ runbook on how to run e2e locally ?  since you have been successfully running them during this massive learning experience, It might be the best time to do a brain dump on all the tips and tricks you discovered when comes time to running e2e on your laptop!